cylc/flow/scripts/lint.py

```diff
@@ -533,6 +572,14 @@ def list_wrapper(line: str, check: Callable) -> Optional[Dict[str, str]]:
     'S013': {
         'short': 'Items should be indented in 4 space blocks.',
         FUNCTION: check_indentation
+    },
+    'S014': {
+        'short': 'Run mode is not live: This task will only appear to run.',
+        FUNCTION: re.compile(r'run mode\s*=\s*[^l][^i][^v][^e]$').findall
```

Comment:
Doesn't work for `run mode = workflow` as that's not 4 chars long. Suggest using negative lookahead instead:
```suggestion
        FUNCTION: re.compile(r'run mode\s*=\s*(?!\s*live\b)').findall
```